Cargo.toml

```diff
@@ -66,9 +66,9 @@ serial_test = { workspace = true }
 tempfile = { workspace = true }
 
 [workspace.dependencies]
-astarte-device-sdk = "0.8.1"
-astarte-device-sdk-mock = "0.8.1"
-astarte-message-hub-proto = "0.6.2"
+astarte-device-sdk = { git = "https://github.com/astarte-platform/astarte-device-sdk-rust" }
+astarte-device-sdk-mock = { git = "https://github.com/astarte-platform/astarte-device-sdk-rust" }
+astarte-message-hub-proto = { git = "https://github.com/astarte-platform/astarte-message-hub-proto" }
```

Comment:
Since there are many PR before the release of these deps, [we should pin the rev for the dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#choice-of-commit).